### ISSUE TITLE
Code quality: consolidate xmlns

### ISSUE THREAD

**Issue by Jay-o-Way** [microsoft/WinUI-Gallery#1673]
Just looking at the namespaces for `using:WinUIGallery.ControlPages` for example. Well, I think this screenshot says it all...
![Image](https://github.com/user-attachments/assets/7e3a6a86-721d-4f58-bd13-98c6b3962053)

**Comment by Jay-o-Way** [comment]
Let's decide on the proper namespaces here:

| xmlns: | using: |
| --- | --- |
| local | WinUIGallery |
|  | WinUIGallery.Common |
|  | WinUIGallery.Data |
|  | WinUIGallery.DesktopWap.Controls |
|  | WinUIGallery.ControlPages |
|  | CommunityToolkit.WinUI |
|  | CommunityToolkit.WinUI.Controls |
|  | Microsoft.UI.Xaml.Media |
|  | Microsoft.UI.Xaml.Controls.AnimatedVisuals |
|  | Microsoft.UI.Xaml.Controls?IsApiContractPresent |

P.S. For what is `IsApiContractPresent` needed?

**Comment by marcelwgn** [comment]
Yeah I think consolidating the xmlns namespaces sounds good though I wouldnt say "local" is a fixed name. From my experience, local usually often just refers to the namespace of the control.

Re API Contract: I think this was just missed when moving to WinUI 3 and cleaning up the UWP remnants.

**Comment by Jay-o-Way** [comment]
Thanks. The combination `local` + `WinUIGallery` is actually very common in the repo, that's why I took the liberty to fill it in, as the first example. But if you want it to be something else, let me know. You can edit the above comment if you want.

I think it's a matter of preference to use abbreviations or words for the namespaces. Short words might be easier to decipher.

So, that means the API contract thing can be removed, am I right?

**Comment by Jay-o-Way** [comment]
Idea: we could rename `WinUIGallery.Controls` to (xmlns:internal) `InternalControls`? Given the many other things that use the word "controls" - it's what this app is all about 😆

**Comment by Jay-o-Way** [comment]
Five cases of `xmlns:converters` are left: `WinUIGallery.Converters` versus `CommunityToolkit.WinUI.Converters`

**Comment by marcelwgn** [comment]
Thanks for the ping @Jay-o-Way ! I think we should do the following:
- If the page is in the same namespace, use local for that namespace e.g. a Control in the folder WinUIGallery.Controls uses `local` for that namespace
- Use toolkit for CommunityToolkit
- Otherwise use the last part of the namespace path, e.g. `controls` for `WinUIGallery.Controls`

I dont think anyone has a reference to the root namespace `WinUIGallery` so we are fine with that.

**Comment by Jay-o-Way** [comment]
@marcelwgn I'm sorry, that comment isn't really helping. Please check the above comment again? "Community Toolkit" is not _one_ thing; it may have a dew sub-divisions. And (e.g.) CT.Converters is ambiguous with WinUIGallery.Converters.

**Comment by marcelwgn** [comment]
In that case, I think we should use the "last namespace" rule but preface it with "toolkit-" so CT.Converters would become "toolkit-converters"
